### PR TITLE
Fixed corner radius on the query button on the AutoSuggestBox control

### PR DESCRIPTION
**Pull request by yaichenbaum**
## Description
I fixed the query button on the AutoSuggestBox having sharp corners. See the before and after photo.

## Motivation and Context
I wanted to fix an inconsistency with the design of the AutoSuggestBox control.

## How Has This Been Tested?
I tested this with the MUXControlsTestApp.

### Before
image
### After
image

## Additional context
Sorry for the typo in the commit.

### CODE REVIEW

**Comment by ranjeshj** [comment]
@chigy as FYI

**Comment by ranjeshj** [comment]
@yaichenbaum  can you please merge with master which includes some stability fixes for the PR pipeline ?

**Comment by yaichenbaum** [comment]
@ranjeshj Done.

**Comment by kmahone** [comment]
Looks like there were some test failures in this run:

>ApiTests.NavigationViewTests.VerifyVisualTree
>ApiTests.AutoSuggestBoxTests.VerifyVisualTree

**Comment by StephenLPeters** [comment]
@yaichenbaum Will you be able to address the visual verification test failures here?

**Comment by yaichenbaum** [comment]
@StephenLPeters I am not sure how I would do that, can I let a little direction on that?

**Comment by chingucoding** [comment]
@yaichenbaum You can find instructions on how to update the Visual Tree Verification files here:
https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/developer_guide.md#visual-tree-verification-tests

**Comment by yaichenbaum** [comment]
@chingucoding Thank you for the link, I will try to get to it over the next few days.

**Comment by yaichenbaum** [comment]
@StephenLPeters I added the new files, can you run the tests again?

**Comment by StephenLPeters** [comment]
@yaichenbaum there are two more visual tree verification masters that need updating AppBarButton-5.xml and AppBarToggleButton-5.xml.  @ranjeshj or @kmahone I don't think I would expect this change to be affecting AppBarButton visuals... thoughts?

File: test/MUXControlsTestApp/master/AppBarToggleButton.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
       FocusVisualPrimaryThickness=2,2,2,2
       FocusVisualPrimaryBrush=#FF000000
       Visibility=Visible
-      RenderSize=68,56
```
Comment (StephenLPeters) [inline test/MUXControlsTestApp/master/AppBarToggleButton.xml:29]: >68,56 [](start = 17, length = 5)

Its possible this file just needs to be reverted. I don't think your change would have caused appBarTOggleButton to be changed

Comment (yaichenbaum) [reply test/MUXControlsTestApp/master/AppBarToggleButton.xml:29]: I am happy to revert the file, just let me know if that is what I should do.

Comment (StephenLPeters) [reply test/MUXControlsTestApp/master/AppBarToggleButton.xml:29]: please revert this, then i'll rerun

Comment (yaichenbaum) [reply test/MUXControlsTestApp/master/AppBarToggleButton.xml]: I reverted the changes to this file, should I do that for any other files as well?

**Comment by yaichenbaum** [comment]
Why is the test still failing?

**Comment by ranjeshj** [comment]
@yaichenbaum Can you please merge in master and I can kick off a new run ? There have been a handful of changes with NavigationView in the past couple of weeks. Thanks!

**Comment by yaichenbaum** [comment]
@ranjeshj Sure.

**Comment by kmahone** [comment]
The test failures on RS4 and RS3 were the result of xml master comparison. NavigationViewTopPaneContent-5.xml and NavigationViewTopPaneContent-6.xml needed to be updated to account for CornerRadius. I've pushed fixes and re-run the build.

**Pull request merged**